share/src/main/java/io/zensoft/share/service/facebook/FacebookPublisherService.java

```diff
@@ -76,14 +60,6 @@ private boolean isValidImageUrl(String imageUrl) {
         return image != null;
```

Comment:
are you sure that url is invalid it's `null`.
i would check if url string is not empty like `imageUrl != null && !imageUrl.replaceAll("\\s","").equals("")`
but it would be amazing to check image availability first, to not catch `exceptions` during runtime in case image is unavailable

Reply:
Good point. However, you would be convinced that there is no problem if you have a look at the whole method, not only at the last line:
private boolean isValidImageUrl(String imageUrl) {
        Image image;
        try {
            image = ImageIO.read(new URL(imageUrl));
        } catch (Exception e) {
            return false;
        }
        return image != null;
    }

Reply:
sorry dude, didn't notice it ;)